build.gradle.kts

```diff
@@ -18,9 +18,9 @@ plugins {
     id("org.sonarqube").version("3.0")
     `maven-publish`
     id("io.spring.dependency-management") version "1.0.10.RELEASE"
-    kotlin("jvm").version("1.4.0")
-    kotlin("plugin.spring").version("1.4.0")
-    kotlin("kapt").version("1.4.0")
+    kotlin("jvm").version("1.4.30")
```

Comment:
Is there a reason why we don't use the Versions.KOTLIN_VERSION for the plugins?

Reply:
no